src/internal/modules/account-entry/controller.js

```diff
@@ -0,0 +1,170 @@
+'use strict';
+
+const { omit } = require('lodash');
+
+const forms = require('shared/lib/forms');
+const routing = require('./lib/routing');
+const session = require('./lib/session');
+
+const { handleFormRequest } = require('shared/lib/form-handler');
+const { NEW_ACCOUNT } = require('./lib/constants');
```

Comment:
This constant is only used in one place (this file) so it might be worth considering removing the `./lib/constants` file and just defining it here. (not important)

Reply:
This constant is also used by the "select existing account" form